l10n_it_fatturapa_in/models/account.py

```diff
@@ -15,6 +16,43 @@ class AccountInvoice(models.Model):
         "einvoice.line", "invoice_id", string="Lines Detail",
         readonly=True, copy=False)
 
+    e_invoice_amount_untaxed = fields.Monetary(
+        string='E-Invoice Untaxed Amount', readonly=True)
+    e_invoice_amount_tax = fields.Monetary(string='E-Invoice Tax Amount',
+                                           readonly=True)
+    e_invoice_amount_total = fields.Monetary(string='E-Invoice Total Amount',
+                                             readonly=True)
+
+    e_invoice_validation_error = fields.Boolean(
```

Comment:
Mi stavo chiedendo se non sarebbe meglio rinominarlo in modo più coerente con il metodo specificato in `compute`.
Inoltre `default=False` mi pare ridondante.

Reply:
@tafaRU Si forse è meglio. Qualcosa tipo `_check_einvoice_amount_equality` o `_check_einvoice_amount_coherence`?

Reply:
A fronte di una review più approfondita non cambierei il nome del campo ma mi limiterei a modificarlo come segue:

```
e_invoice_validation_error = fields.Boolean(compute='_compute_e_invoice_validation_error`)
```
in linea con [le convenzioni OCA](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#models): `Compute Field: the compute method pattern is _compute_<field_name>`.

Il `default=False` dovrebbe essere ridondante.

Reply:
@tafaRU Fatto! Ho appena pushato le modifiche. Grazie Alex per la review.

Reply:
Grazie a te per la PR :+1: Ho aggiunto https://github.com/OCA/l10n-italy/pull/1144/files#r272190717